src/main/java/io/cryostat/net/reports/ActiveRecordingReportCache.java

```diff
@@ -109,14 +119,7 @@ Future<String> get(
     }
 
     boolean delete(ConnectionDescriptor connectionDescriptor, String recordingName) {
-        RecordingDescriptor key = new RecordingDescriptor(connectionDescriptor, recordingName);
-        boolean hasKey = cache.asMap().containsKey(key);
-        if (hasKey) {
-            logger.trace("Invalidated active report cache for {}", recordingName);
-            cache.invalidate(key);
-        } else {
-            logger.trace("No cache entry for {} to invalidate", recordingName);
-        }
+        boolean hasKey = invalidateCacheEntry(connectionDescriptor, recordingName);
         return hasKey;
```

Comment:
It doesn't look like there is any meaningful difference between `delete()` and the new `invalidateCacheEntry()` methods, is there? The implementation was just moved from the first to the second, and all the first does now is call the second and return its return value.